api/src/main/java/com/thesurvey/api/exception/ErrorMessage.java

```diff
@@ -24,7 +24,10 @@ public enum ErrorMessage {
     NO_ONLY_WHITESPACE("공백만 입력할 수 없습니다."),
     CREATOR_CANNOT_ANSWER("설문조사 작성자는 자신의 설문조사에 응답할 수 없습니다."),
     NOT_SURVEY_QUESTION("해당 설문조사의 질문이 아닙니다."),
-    INVALID_REQUEST("유효하지 않은 요청입니다");
+    INVALID_REQUEST("유효하지 않은 요청입니다"),
+    INVALID_PAGE_SIZE("페이지 크기는 8 이어야 합니다."),
+    INVALID_PAGE_SORT("페이지 정렬은 기본 값이어야 합니다."),
```

Comment:
Same above. Client should not be able to set anything for pagination except request parameter `page` which indicates actual page number. Seems like this error message is for some client request that could change the method of sorting policy

Reply:
Oh okay this is  for **filtering** option. Resolved.